lib/editor-parser.ts

```diff
@@ -3,16 +3,24 @@ import { ReviewTaskMachineContextReviewData } from "../server/claim-review-task/
 import { ReportModelEnum } from "../server/types/enums";
 
 type SchemaType = {
-    summary: string;
+    summary?: string;
     verification?: string;
     report?: string;
-    sources: any[];
+    sources?: any[];
```

Comment:
suggestion: create 2 different schema types and make the Schema Type 1 or the other. The 2 new schema types should have the same base schema.

That way you can avoid having sources and source in the same schema type.